superset/commands/importers/v1.py

```diff
@@ -0,0 +1,113 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from typing import Any, Dict
+
+import yaml
+from marshmallow import fields, Schema, validate
+from marshmallow.exceptions import ValidationError
+from sqlalchemy.orm import Session
+
+from superset import db
+from superset.commands.base import BaseCommand
+from superset.commands.exceptions import CommandInvalidError
+from superset.models.helpers import ImportExportMixin
+
+METADATA_FILE_NAME = "metadata.yaml"
+IMPORT_VERSION = "1.0.0"
```

Comment:
do you have a process on when this would be upgraded to 1.0.1 vs. 2.0.0

Reply:
The idea is that if we bump the format to, say, `1.0.1` we would change the validation to accept anything between `1.0.0` and `1.0.1`, and adjust the class to handle the differences. If we bump it to `2.0.0` we should right a new command under `v2`, so that we can still be able to import `1.0.0` with the `v1` command.